apps/layout-test/src/pages/main/main.component.html

```diff
@@ -1,12 +1,10 @@
-<h1>
-	NgxTour
-</h1>
+<h1>NgxTour</h1>
 
 <button (click)="startTour()">
-	{{dataString$ | async}}
+	{{ dataString$ | async }}
 </button>
 
-<h1 tourItem="start">NgxDisplayContent</h1>
+<h1 tourItem="start" contenteditable="true">NgxDisplayContent</h1>
```

Comment:
Why was this added?

Reply:
I added this to show that the body class with `pointer-events: none` is properly set onto the body only when the tour is active, as can be seen in the screenrecording.
I found this the easiest way to showcase this without adding another tour step. So just laziness, I guess? 🤷